src/System.IO.UnmanagedMemoryStream/tests/UmsTests.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using Xunit;
+using System.Threading;
 
 // TODO: add WriteAsync, Timeout, Flush, CopyTo tests
```

Comment:
I meant: could you add a comment (at the top of this file) stating which APIs are not covered by these tests and why? Possibly it's just a simple comment about CopyTo.